### PR TITLE
release-schema: Add missing eForms fields

### PR DESCRIPTION
**Pull request by duncandewhurst**
Per https://github.com/open-contracting/european-union-support/issues/186#issuecomment-1659444776

### CODE REVIEW

File: release-schema.json [hunk 1]
```diff
@@ -159,6 +168,32 @@
...
+          "description": "The rationale for not including accessibility criteria despite the object of the lot being intended for use by natural persons.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "minLength": 1
+        },
+        "reviewDetails": {
+          "title": "Review details",
+          "description": "Details of the review procedures for the lot, such as the the deadline for review requests.",
```
Comment (jpmckinney) [inline release-schema.json]: `Tender/reviewDetails` and `Lot/reviewDetails` should be aligned.

File: release-schema.json [hunk 2]
```diff
@@ -159,6 +168,32 @@
           "title": "Award period",
           "description": "The period for decision making regarding the contract award for the lot. The end date should be the date on which an award decision is due to be finalized. The start date may be used to indicate the start of an evaluation period.",
           "$ref": "#/definitions/Period"
+        },
+        "hasAccessibilityCriteria": {
+          "title": "Has accessibility criteria",
+          "description": "Whether the technical specifications for the lot include criteria for the accessibility of disabled persons.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "description": "Whether the technical specifications for the lot include accessibility criteria for people with disabilities.",
```

File: release-schema.json [hunk 3]
```diff
@@ -159,6 +168,32 @@
...
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "noAccessibilityCriteriaRationale": {
+          "title": "No accessibility criteria rationale",
+          "description": "The rationale for not including accessibility criteria despite the object of the lot being intended for use by natural persons.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "description": "The rationale for not including accessibility criteria when the object of the lot is intended for use by natural persons.",
```

File: release-schema.json [hunk 4]
```diff
@@ -56,6 +56,15 @@
               "minLength": 1
             }
           }
+        },
+        "eDeliveryGateway": {
+          "title": "eDelivery gateway",
+          "description": "The organization's URL for the exchange of data and documents.",
```
Comment (jpmckinney) [inline release-schema.json]: https://ec.europa.eu/digital-building-blocks/wikis/display/DIGITAL/eDelivery

```suggestion
          "description": "The organization's URL for the exchange of data and documents with eDelivery.",
```

File: release-schema.json (hunk 1 above)
Comment (duncandewhurst) [reply release-schema.json]: Ah, I didn't realise we already had the field on `Tender`. Fixed in https://github.com/open-contracting-extensions/ocds_eu_extension/pull/14/commits/1f4fad3d82e54146d478e75c6f140faadab8267d

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**